tensorflow/contrib/gan/python/eval/python/classifier_metrics_impl.py

```diff
@@ -66,6 +67,7 @@
 
 
 INCEPTION_URL = 'http://download.tensorflow.org/models/frozen_inception_v1_2015_12_05.tar.gz'
+INCEPTION_TAR = 'frozen_inception_v3_2017_09_13.tar.gz'  # location of temporary download file
```

Comment:
These filenames are mismatched. Probably better just to split on the last / and use that as the tmp file

Reply:
@andrewharp so now I'm wondering where I got that filename from. I'll make the fix you suggested.

I confirmed the original paper (https://arxiv.org/pdf/1606.03498.pdf) uses the 2015v1 model, so probably that one should be the default. However, it is not unreasonable to compute an inceptionv3 score.

Maybe I'll add an additional set of functions like `inception_score_v3` (separate PR).

Reply:
removed the extra constant. `_default_graph_def_fn` now uses `os.path.basename(INCEPTION_URL)`.